content/05-gh-actions.md

```diff
@@ -121,58 +122,66 @@ jobs:
         pytest example.py
 ```
 
+Commit the change by pressing the "Start Commit" button:
 
-Do the commit by pressing  the "Start Commit" button:
-
-
-<img src="{{ site.baseurl}}/img/gh_action_commit.png"/>
+```{figure} img/gh_action_commit.png
+:alt: Committing the change
+:width: 400px
+```
 
----
 
 ### Step 4: Verify that tests have been automatically run
 
 Observe in the repository how the test succeed. While the test is
 executed, the repository has yellow marker. This is replaced with a green
 check mark, once the test succeeds:
 
-<img src="{{ site.baseurl}}/img/green_check_mark.png"/>
+```{figure} img/green_check_mark.png
+:alt: Verify that the test passed
+```
```

Comment:
````suggestion
```{figure} img/green_check_mark.png
:alt: Verify that the test passed

Green check means passed.
```
````